src/styles.css

```diff
@@ -180,3 +180,62 @@ body {
   border: 0;
   margin-left: 10px;
 }
+
+#empty-pokemon-details {
+  box-sizing: border-box;
```

Comment:
If you find yourself repeating similar styles, you can move them into a higher scope (this is the _cascading_ in _Cascading Style Sheets_). So if you want to use `box-sizing: border-box` everywhere, you can add it to the `html` and `body` selectors:

```css
html, body {
  box-sizing: border-box;
}
```

Reply:
Still learning best practices here, thanks!